pandas/util/testing.py

```diff
@@ -1964,6 +1964,32 @@ def add_nans_panel4d(panel4d):
     return panel4d
 
 
+def gen_zeros(arr_len):
```

Comment:
move this to conftest as its just a fixture

Reply:
This is going to be re-used in Series tests a few PRs from now.  Which conftest should it go into?

Reply:
you can put it in the top-level one pandas/conftest.py I would really like to move more things there

Reply:
How do we pass the `arr_len` arg to it?

Reply:
you don't you move this as a function, then create as a fixture.

Reply:
So I move the function to conftest, then from test_numeric create a fixture like (?):
```
@pytest.fixture(params=[x for x in gen_zeros(5)
                                          if not isinstance(x, Series)])
def zero(request):
    # For testing division by (or of) zero for Series with length 5, this
    # gives several scalar-zeros and length-5 vector-zeros
    return request.param
```

Do we import conftest?  I haven't seen that elsewhere.  What's the upside of this over just having `zeros` in the module namespace?

Reply:
yes. you don't need to import conftest. you can put this in the module only if its *only* used there. the point of conftest is to share.

Reply:
since this is only used in 1 place, just move it to the test file itself and make it into a fixture